404.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Нёрдс</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a>
+          <img src="https://via.placeholder.com/150x100" alt="">
+        </a>
+        <ul>
+          <li><a="404.html">Студия</a></li>
```

Comment:
Так, тут везде был какой-то странный реплейс (видимо делала в редакторе автоматическим replace, но не проверила КАК он заменил, или немного некорректно ввела в строках что на что заменить)

Почти у всех `<a>` пропали `href`

```suggestion
          <li><a href="404.html">Студия</a></li>
```